feincms/templates/admin/content/richtext/init_tinymce4.html

```diff
@@ -48,7 +48,7 @@
             $('{% block selectors %}.order-machine textarea.item-richtext{% endblock %}').each(function(){
                 feincms_richtext_add_tinymce(this);
             });
-        }
+        };
```

Comment:
Maybe you could convert this to `function richtext_init_fn() { ... }` ; I don't think there's a good reason for this to use the `var richtext_init_fn = function() { ... }` pattern. Does this work too in your local environment?

Reply:
Sure, no problem, done. It does.